circle.yml

```
@@ -536,6 +536,11 @@ jobs:
       - run:
           name: Test
           command: cargo test
+      - run:
+          name: Package
+          command: |
```

Fancy way of doing this.
```suggestion
          working_dir: bindings/rust
          command: cargo package
```

---

How can I leave a comment in that case? Just as a yaml comment?

---

Yes. YAML commands are better IMHO.

---

Is `~/bindings/rust` the current path? The suggested one does not work.